src/ck-ldb/HybridBaseLB.C

```diff
@@ -106,11 +106,11 @@ void HybridBaseLB::initTree()
 HybridBaseLB::~HybridBaseLB()
 {
 #if CMK_LBDB_ON
-  theLbdb = CProxy_LBDatabase(_lbdb).ckLocalBranch();
-  if (theLbdb) {
-    theLbdb->
-      RemoveNotifyMigrated(notifier);
-    //theLbdb->
+  lbmgr = CProxy_LBManager(_lbmgr).ckLocalBranch();
+  if (lbmgr) {
+//    lbmgr->
```

Comment:
delete

Reply:
Also, I'm noticing a ton of this code seems very duplicated between the different base load balancers (Central, DistBase, HybridBase). Should this common stuff be moved to a base class? Or in the subsequent changes which get rid of CentralLB and add TreeLB etc, maybe its no longer relevant?

Reply:
TreeLB should take care of most of it.